ram/hram.asm

```diff
@@ -330,8 +330,7 @@ hGymTrashCanRandNumMask::
 	db
 
 NEXTU
-hFFDB:: db
-hFFDC:: db
+hWasBookShelfTextFound:: db ; set to 0 if found, $FF if not found when attempting to interact with a hidden object
```

Comment:
Assuming that "found" means what I think it means here (the comment restates the label name), and since bookshelves aren't the only hidden objects:
```suggestion
hTalkingToHiddenObject:: db ; set to 0 if found, $FF if not found when attempting to interact with a hidden object
```

Reply:
the value is specifically set only when running the bookshelf code... 0 if you interacted with a bookshelf, $ff if you didn't